src/los_client/cli.py

```diff
@@ -0,0 +1,158 @@
+                    self.output_path = config_json.get("output_path")
+                    self.problem_path = config_json.get("problem_path")
+            except (json.JSONDecodeError, IOError):
+                print("Warning: Failed to load configuration."
+                      " Creating new one.")
+                self.save_config()
+        else:
+            print("No existing configuration found. Creating new one.")
+            self.save_config()
+
+    def save_config(self) -> None:
+
+        config_json = {
+            "solver_path": self.solver_path,
+            "output_path": self.output_path,
+            "problem_path": self.problem_path,
+        }
+        try:
+            with open(self.json_path, "w") as config_file:
+                json.dump(config_json, config_file)
+        except IOError as e:
+            print(f"Error saving configuration: {e}", file=sys.stderr)
+
+@dataclass
+class SatCLI:
+    output_folder = os.path.join(os.path.dirname(__file__) , "output")
+    config : CLIConfig
+
+    def __init__(self) -> None:
+        self.config = CLIConfig(os.path.join(
+            self.output_folder, "config.json"))
+        self.config.load_config()
+
+    def configure(self, args:Any) -> None:
+
+        if args.solver:
+            if isinstance(args.solver, Dummy):
+                print("Current path to solver is: ", (self.config.solver_path
+                            if self.config.solver_path!="" else "No path set"))
+            else:
+                self.config.solver_path = args.solver
+                print(f"Solver path set to: {self.config.solver_path}")
+                self.config.save_config()
+
+        if args.output:
+            if isinstance(args.output, Dummy) :
+                print("Current path to output file is: ", self.config.
+                output_path if self.config.output_path!="" else "No path set")
+            else:
+                self.config.output_path = args.output
+                print(f"Output path set to: {self.config.output_path}")
+                self.config.save_config()
+
+        if args.problem:
+            if isinstance(args.problem, Dummy):
+                print("Current path to problem file is: ", self.config.
+                problem_path if self.config.problem_path!=""else "No path set")
+            else:
+                self.config.problem_path = args.problem
+                print(f"Problem path set to: {self.config.problem_path}")
+                self.config.save_config()
+
+    def confirm(self) -> None:
+        if not (self.config.solver_path and self.config.output_path
+                and self.config.problem_path):
+            print("Error: Please provide all paths (-path, -output, -problem) "
+                  "before confirmation.")
+            return
+        # TODO: Ensure the solver script is executable
+        if False:
+            try:
+                result = subprocess.run(
```

Comment:
You will need https://docs.python.org/3/library/asyncio-subprocess.html#asyncio.create_subprocess_exec because we are using asyncio.